Core/src/main/java/org/tribuo/util/Util.java

```diff
@@ -1040,4 +1055,36 @@ public static String formatDuration(long startMillis, long stopMillis) {
         return diffIndicesList.stream().mapToInt(Integer::intValue).toArray();
     }
 
+    /**
+     * Standardizes the input, i.e. divides it by the variance and subtracts the mean.
```

Comment:
The i.e. comment here is weird. Should say something like "a value is standardized by subtracting the mean and dividing the result by the variance". The way it's stated sounds like we divide first and then subtract, which is incorrect (assuming the code is correct, which I think it is.)

standardizeInPlace javadoc has the same problem.

I guess I'm curious why these methods are static when we have class members for doing the accumulation? Are they used elsewhere? If they are and staticness makes sense for these, then there should be non-static ones that use the mean and variance that we've accumulated on an input array.

Reply:
I added class members to the `MeanVarianceAccumulator` which performs standardization by calling into `Util.standardize`. The `MeanVarianceAccumulator` didn't end up being used by the standardization code in `LibSVMRegressionTrainer`, but my plan is to migrate all other the places where I do this over to this code over time, so I put it into this PR along with it's test.